release-notes/cloudflow-1.3.1.md

```diff
@@ -0,0 +1,42 @@
+# Cloudflow 1.3.1 Release Notes
+
+Cloudflow 1.3.1 is a minor patch release and contains the following changes:
```

Comment:
```suggestion
Cloudflow 1.3.1 is a patch release and contains the following changes:
```

Reply:
This is not a minor release IMO, we upgrade a lot of components, as a customer I would be very interested in why and what this means.